Jenkinsfile

```diff
@@ -35,6 +35,7 @@ pipeline {
         GITHUB_USERNAME = 'codice'
         GITHUB_TOKEN = credentials('cxbot')
         GITHUB_REPONAME = 'alliance'
+        DOCKERHUB_CREDS = 'dockerhub-codicebot'
```

Comment:
Where is this variable being used?

Reply:
see DOCKERHUB_CREDS_USR and DOCKERHUB_CREDS_PSW, are automatically generated from that variable
https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#usernames-and-passwords

Reply:
Wow, I did not know about this! Thanks for explaining it to me @TonyMorrison!